### PR TITLE
TELCODOCS-1819: Rename controller-manager to controller

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
D/S Docs: [MGMT-17316] Doc updates for v2.x 

Jira: https://issues.redhat.com/browse/MGMT-17316 (Rename controller-manager to controller)

Version(s): KMM 2.1,  openshift-4.15, openshift-4.14 

Link to docs preview: https://73714--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management

SME review: @qbarrand
QE review: @cdvultur

### CODE REVIEW

**Comment by StephenJamesSmith** [comment]
@qbarrand @cdvultur Please review to ensure that all instances of **controller-manager** have been changed to **controller**. Please add /lgtm comment if all is well. Thx!

**Review by cdvultur — APPROVED** [review]
/lgtm

File: modules/kmm-gathering-data-for-kmm.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Module included in the following assemblies:
+/ Module included in the following assemblies:
```
Comment (qbarrand) [inline modules/kmm-gathering-data-for-kmm.adoc:1]: Is this desirable?

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by JoeAldinger** [comment]
@StephenJamesSmith does this go into 4.16 too?

**Comment by StephenJamesSmith** [comment]
@JoeAldinger Based on the Epic and thee comments from @qbarrand (_Let us put all the new content in 4.14+. We will likely release KMM 2.1 (April / May) for OCP 4.12+. At that point we will ask to align all 4.1{2,3,4} docs on whatever we have for 4.15.)_ I assume it will end up in 4.16.

**Comment by JoeAldinger** [comment]
> issues.redhat.com/browse/MGMT-17316

4.14+ I would think includes 4.16. The reason I'm asking is that it would be an edge case to put something in 4.14 and 4.15, but not 4.16. Not that, that isn't what this is. I just want to make sure we get this in the right branches.

**Pull request merged**